src/app/api/course/create.ts

```diff
@@ -1,15 +1,12 @@
-import SPARKED_PROCESS_CODES from "app/shared/processCodes";
```

Comment:
I think if we are using one formatting too it will help us to avoid seeing changes like these .